src/elements/hero/src/index.tsx

```diff
@@ -55,92 +56,97 @@ const Hero: React.FC<Props> = ({
+        sx={{ variant: 'elements.hero.wrapper', overflow: 'hidden' }}
+        px={{ backgroundColor: 'backgroundColor' }}
       >
-        <Container>
-          {breadcrumbs && (
-            <div
-              sx={{ paddingTop: 'sm', variant: 'elements.hero.breadcrumbs' }}
-            >
-              {breadcrumbWithVariant}
-            </div>
-          )}
-          <div
-            sx={{
-              position: 'relative',
-              display: 'block',
-              flexDirection: 'row-reverse'
-            }}
-          >
-            <div
-              sx={{
-                position: 'absolute',
-                left: [0, '45%'],
-                right: 0,
-                top: 0,
-                bottom: 0,
-                display: [
-                  fgImageOnMobile ? 'block' : 'none',
-                  fgImageOnTablet ? 'block' : 'none',
-                  'block'
-                ],
-                ...(fgImage && fgImageType === 'background'
-                  ? {
-                      backgroundImage: `url(${fgImage})`,
-                      backgroundRepeat: 'no-repeat',
-                      ...fgImagePosition
-                    }
-                  : undefined),
-                variant: 'elements.hero.image'
-              }}
-            >
-              {fgImage && fgImageType === 'img' && (
-                <img
-                  sx={{
-                    maxWidth: '100%',
-                    position: 'absolute',
-                    ...fgImagePosition
-                  }}
-                  src={fgImage}
-                  role="presentation"
-                />
-              )}
-            </div>
+        <div
+          sx={{
+            position: 'relative',
+            overflow: 'hidden',
+            backgroundColor: customBgColor || undefined
+          }}
+          className={className}
+        >
+          <Container>
+            {breadcrumbs && (
+              <div
+                sx={{ paddingTop: 'sm', variant: 'elements.hero.breadcrumbs' }}
+              >
+                {breadcrumbWithVariant}
+              </div>
+            )}
             <div
               sx={{
                 position: 'relative',
-                paddingTop: ['sm', 'xxl'],
-                paddingBottom: ['sm', 'xxl'],
-                variant: 'elements.hero.content'
+                display: 'block',
+                flexDirection: 'row-reverse'
               }}
             >
-              {children}
+              <div
+                sx={{
+                  position: 'absolute',
+                  left: [0, '45%'],
+                  right: 0,
+                  top: 0,
+                  bottom: 0,
+                  display: [
+                    fgImageOnMobile ? 'block' : 'none',
+                    fgImageOnTablet ? 'block' : 'none',
+                    'block'
+                  ],
+                  ...(fgImage && fgImageType === 'background'
+                    ? {
```

Comment:
should the spread be in front of the object on line 100?

Reply:
I didn't add this, this was from Callum. It doesnt work when I move the spread operator to line 100